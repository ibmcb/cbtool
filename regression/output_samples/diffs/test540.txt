 status: Waiting for vm_47 (cloud-assigned uuid 3FB2C3DE-B897-55FF-84FF-5CB02663880F) to start...
 status: Waiting for vm_50 (cloud-assigned uuid B0B170F5-DF65-51BB-894D-A86FB5AE962A) to start...
 status: Waiting for vm_48 (cloud-assigned uuid 18471E43-6507-5D95-9E5E-6D90007A5179) to start...
 status: Waiting for vm_51 (cloud-assigned uuid 2DE61115-47FC-50A1-8028-7857EF277CDF) to start...
 status: Waiting for vm_49 (cloud-assigned uuid 0FB9D486-BCDE-54C9-8F92-F3437B788932) to start...
 status: Trying to establish network connectivity to vm_50 (cloud-assigned uuid B0B170F5-DF65-51BB-894D-A86FB5AE962A), on IP address 222.96.102.122...
 status: Trying to establish network connectivity to vm_48 (cloud-assigned uuid 18471E43-6507-5D95-9E5E-6D90007A5179), on IP address 122.121.182.182...
 status: Trying to establish network connectivity to vm_51 (cloud-assigned uuid 2DE61115-47FC-50A1-8028-7857EF277CDF), on IP address 77.50.44.100...
 status: Trying to establish network connectivity to vm_49 (cloud-assigned uuid 0FB9D486-BCDE-54C9-8F92-F3437B788932), on IP address 251.55.165.89...
 status: Trying to establish network connectivity to vm_47 (cloud-assigned uuid 3FB2C3DE-B897-55FF-84FF-5CB02663880F), on IP address 233.224.62.106...
 status: Bypassing the sending of a copy of the code tree to vm_51 (77.50.44.100)...
 status: Bypassing the sending of a copy of the code tree to vm_50 (222.96.102.122)...
 status: Bypassing the sending of a copy of the code tree to vm_48 (122.121.182.182)...
 status: Bypassing the sending of a copy of the code tree to vm_47 (233.224.62.106)...
 status: Bypassing the sending of a copy of the code tree to vm_49 (251.55.165.89)...
AI object 060277C4-6B56-56A6-AA93-D19560569E69 (named "ai_8") sucessfully attached to this experiment. It is ssh-accessible at the IP address 222.96.102.122 (B0B170F5-DF65-51BB-894D-A86FB5AE962A.simcloud.com).
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |16.174.212.90   |1374527148     |none                                    |FC7A6FC0-808A-55B4-A80F-E9F94F7EE53E
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |166.31.137.41   |1374527189     |none                                    |FBE84899-B486-5087-8F5D-6F505D042585
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |116.80.208.127  |1374527207     |none                                    |F7A40D5B-B8DF-503D-BFC2-6C58EDD596F2
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |222.96.102.122  |1374527295     |none                                    |060277C4-6B56-56A6-AA93-D19560569E69
|vm_33         |client_daytrader    |nano32      |16.174.212.90   |simhostd3                  |LG                  |ai_3    |none       |D6DEE6FD-5C09-5F1E-9F0F-8327CEBE7A47
|vm_34         |db2                 |copper32    |19.206.141.143  |simhostb3                  |SUT                 |ai_3    |none       |2726062C-1A5E-504D-AAE6-A27DC8B77B31
|vm_35         |was                 |iron32      |206.54.3.178    |simhostb3                  |SUT                 |ai_3    |none       |154EBE1E-6C2C-52AB-B17E-FDA0E360EDE3
|vm_36         |was                 |iron32      |69.161.136.59   |simhostb0                  |SUT                 |ai_4    |none       |29235B2B-44B0-5233-8155-84A65D813CF7
|vm_37         |client_daytrader    |nano32      |166.31.137.41   |simhostd3                  |LG                  |ai_4    |none       |E6421AAA-785B-5D3E-9327-C34590446358
|vm_38         |db2                 |copper32    |3.141.157.45    |simhostc2                  |SUT                 |ai_4    |none       |852EF342-A897-5208-B040-B6BAA88EF671
|vm_40         |netserver           |iron32      |10.50.107.94    |simhostc1                  |SUT                 |ai_5    |none       |3599B3EB-9B41-53FC-B468-846C40D223F3
|vm_39         |netclient           |platinum64  |116.80.208.127  |simhostc1                  |SUT                 |ai_5    |none       |13A46737-D784-5408-928A-0BD421B69AB9
|vm_48         |was                 |iron32      |122.121.182.182 |simhostb2                  |SUT                 |ai_8    |none       |FA6A662D-695E-5685-AF59-02582954F495
|vm_47         |lb                  |iron32      |233.224.62.106  |simhosta1                  |SUT                 |ai_8    |none       |B4F1652E-5A0B-52C0-B862-62E490F47277
|vm_50         |client_daytrader    |nano32      |222.96.102.122  |simhostd1                  |LG                  |ai_8    |none       |18C8F60B-A31F-5C1D-A7E4-73265F651EC7
|vm_49         |was                 |iron32      |251.55.165.89   |simhosta4                  |SUT                 |ai_8    |none       |6384A6E1-615D-59E2-8DED-9DDEEB10B6D2
|vm_51         |db2                 |gold32      |77.50.44.100    |simhostc0                  |SUT                 |ai_8    |none       |9B830FEB-2DAA-5A28-AD05-BBE5B3CF975B
|Used Memory                                          |41.05M                        
|Uptime (in seconds)                                  |13566075                      
|Total Connections Received                           |6495057                       
|Total Commands Processed                             |94034364                      
|Number of Keys                                       |544                           
|Storage Size                                         |16113934320.0                 
|Data Size                                            |13823471056.0                 
|Index Size                                           |550481904                     
|Average Object Size                                  |6600.73843621                 
|Collections                                          |60                            
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|3                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|2                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|4                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|4                             
