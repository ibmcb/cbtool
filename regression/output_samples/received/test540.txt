 status: Waiting for vm_47 (cloud-assigned uuid 3FB2C3DE-B897-55FF-84FF-5CB02663880F) to start...
 status: Waiting for vm_50 (cloud-assigned uuid B0B170F5-DF65-51BB-894D-A86FB5AE962A) to start...
 status: Waiting for vm_48 (cloud-assigned uuid 18471E43-6507-5D95-9E5E-6D90007A5179) to start...
 status: Waiting for vm_51 (cloud-assigned uuid 2DE61115-47FC-50A1-8028-7857EF277CDF) to start...
 status: Waiting for vm_49 (cloud-assigned uuid 0FB9D486-BCDE-54C9-8F92-F3437B788932) to start...
 status: Trying to establish network connectivity to vm_50 (cloud-assigned uuid B0B170F5-DF65-51BB-894D-A86FB5AE962A), on IP address 222.96.102.122...
 status: Trying to establish network connectivity to vm_48 (cloud-assigned uuid 18471E43-6507-5D95-9E5E-6D90007A5179), on IP address 122.121.182.182...
 status: Trying to establish network connectivity to vm_51 (cloud-assigned uuid 2DE61115-47FC-50A1-8028-7857EF277CDF), on IP address 77.50.44.100...
 status: Trying to establish network connectivity to vm_49 (cloud-assigned uuid 0FB9D486-BCDE-54C9-8F92-F3437B788932), on IP address 251.55.165.89...
 status: Trying to establish network connectivity to vm_47 (cloud-assigned uuid 3FB2C3DE-B897-55FF-84FF-5CB02663880F), on IP address 233.224.62.106...
 status: Bypassing the sending of a copy of the code tree to vm_51 (77.50.44.100)...
 status: Bypassing the sending of a copy of the code tree to vm_50 (222.96.102.122)...
 status: Bypassing the sending of a copy of the code tree to vm_48 (122.121.182.182)...
 status: Bypassing the sending of a copy of the code tree to vm_47 (233.224.62.106)...
 status: Bypassing the sending of a copy of the code tree to vm_49 (251.55.165.89)...
 status: Bypassing generic VM post_boot configuration on all VMs beloging to ai_8...
 status: Bypassing application-specific configuration on all VMs beloging to ai_8...
AI object 060277C4-6B56-56A6-AA93-D19560569E69 (named "ai_8") sucessfully attached to this experiment. It is ssh-accessible at the IP address 222.96.102.122 (B0B170F5-DF65-51BB-894D-A86FB5AE962A.simcloud.com).
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |16.174.212.90   |1374527148     |none                                    |FC7A6FC0-808A-55B4-A80F-E9F94F7EE53E
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |166.31.137.41   |1374527189     |none                                    |FBE84899-B486-5087-8F5D-6F505D042585
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |116.80.208.127  |1374527207     |none                                    |F7A40D5B-B8DF-503D-BFC2-6C58EDD596F2
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |222.96.102.122  |1374527295     |none                                    |060277C4-6B56-56A6-AA93-D19560569E69

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |nano32      |16.174.212.90   |simhostd3                  |LG                  |ai_3    |none       |D6DEE6FD-5C09-5F1E-9F0F-8327CEBE7A47
|vm_34         |db2                 |copper32    |19.206.141.143  |simhostb3                  |SUT                 |ai_3    |none       |2726062C-1A5E-504D-AAE6-A27DC8B77B31
|vm_35         |was                 |iron32      |206.54.3.178    |simhostb3                  |SUT                 |ai_3    |none       |154EBE1E-6C2C-52AB-B17E-FDA0E360EDE3
|vm_36         |was                 |iron32      |69.161.136.59   |simhostb0                  |SUT                 |ai_4    |none       |29235B2B-44B0-5233-8155-84A65D813CF7
|vm_37         |client_daytrader    |nano32      |166.31.137.41   |simhostd3                  |LG                  |ai_4    |none       |E6421AAA-785B-5D3E-9327-C34590446358
|vm_38         |db2                 |copper32    |3.141.157.45    |simhostc2                  |SUT                 |ai_4    |none       |852EF342-A897-5208-B040-B6BAA88EF671
|vm_40         |netserver           |iron32      |10.50.107.94    |simhostc1                  |SUT                 |ai_5    |none       |3599B3EB-9B41-53FC-B468-846C40D223F3
|vm_39         |netclient           |platinum64  |116.80.208.127  |simhostc1                  |SUT                 |ai_5    |none       |13A46737-D784-5408-928A-0BD421B69AB9
|vm_48         |was                 |iron32      |122.121.182.182 |simhostb2                  |SUT                 |ai_8    |none       |FA6A662D-695E-5685-AF59-02582954F495
|vm_47         |lb                  |iron32      |233.224.62.106  |simhosta1                  |SUT                 |ai_8    |none       |B4F1652E-5A0B-52C0-B862-62E490F47277
|vm_50         |client_daytrader    |nano32      |222.96.102.122  |simhostd1                  |LG                  |ai_8    |none       |18C8F60B-A31F-5C1D-A7E4-73265F651EC7
|vm_49         |was                 |iron32      |251.55.165.89   |simhosta4                  |SUT                 |ai_8    |none       |6384A6E1-615D-59E2-8DED-9DDEEB10B6D2
|vm_51         |db2                 |gold32      |77.50.44.100    |simhostc0                  |SUT                 |ai_8    |none       |9B830FEB-2DAA-5A28-AD05-BBE5B3CF975B

The AI object ai_8, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |true                                 
|sut                                     |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |41.05M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13566075                      
|Total Connections Received                           |6495057                       
|Total Commands Processed                             |94034364                      
|Number of Keys                                       |544                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16113934320.0                 
|Data Size                                            |13823471056.0                 
|Index Size                                           |550481904                     
|Average Object Size                                  |6600.73843621                 
|Collections                                          |60                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |13                            
|SVMs                                                 |0                             
|AIs                                                  |4                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |13                            
|VMs ARRIVED                                          |48                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |4                             
|AIs ARRIVED                                          |8                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |186                           

 ---------------- VMC-WIDE COUNTERS ----------------
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|3                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|2                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|4                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|4                             

