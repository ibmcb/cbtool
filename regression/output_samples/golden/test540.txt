 status: Waiting for vm_51 (cloud-assigned uuid 379067A8-3961-5EEF-BD93-ACDBDF96FBB6) to start...
 status: Waiting for vm_47 (cloud-assigned uuid A63B6542-D45C-55A9-9E38-A9931CBCFE3E) to start...
 status: Waiting for vm_49 (cloud-assigned uuid FCF8E34A-1092-510C-A870-498C2F96BD40) to start...
 status: Waiting for vm_48 (cloud-assigned uuid 14A7B418-0EA7-54FF-B10C-53988478ABAB) to start...
 status: Waiting for vm_50 (cloud-assigned uuid 5C605F2B-F86C-5325-9438-31E22FEF528E) to start...
 status: Trying to establish network connectivity to vm_51 (cloud-assigned uuid 379067A8-3961-5EEF-BD93-ACDBDF96FBB6), on IP address 243.57.239.247...
 status: Trying to establish network connectivity to vm_48 (cloud-assigned uuid 14A7B418-0EA7-54FF-B10C-53988478ABAB), on IP address 20.123.111.87...
 status: Trying to establish network connectivity to vm_49 (cloud-assigned uuid FCF8E34A-1092-510C-A870-498C2F96BD40), on IP address 146.189.104.94...
 status: Trying to establish network connectivity to vm_50 (cloud-assigned uuid 5C605F2B-F86C-5325-9438-31E22FEF528E), on IP address 236.129.55.114...
 status: Trying to establish network connectivity to vm_47 (cloud-assigned uuid A63B6542-D45C-55A9-9E38-A9931CBCFE3E), on IP address 49.4.184.107...
 status: Bypassing the sending of a copy of the code tree to vm_47 (49.4.184.107)...
 status: Bypassing the sending of a copy of the code tree to vm_51 (243.57.239.247)...
 status: Bypassing the sending of a copy of the code tree to vm_48 (20.123.111.87)...
 status: Bypassing the sending of a copy of the code tree to vm_50 (236.129.55.114)...
 status: Bypassing the sending of a copy of the code tree to vm_49 (146.189.104.94)...
 status: Bypassing generic VM post_boot configuration on all VMs beloging to ai_8...
 status: Bypassing application-specific configuration on all VMs beloging to ai_8...
AI object E4C12778-5BB2-5169-A366-0AEAB7A196D0 (named "ai_8") sucessfully attached to this experiment. It is ssh-accessible at the IP address 146.189.104.94 (FCF8E34A-1092-510C-A870-498C2F96BD40.simcloud.com).
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |220.74.118.34   |1372702347     |none                                    |896E79BE-2D86-531C-9C3C-5A621DFAA5A7
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |16.152.177.242  |1372702386     |none                                    |BC1F9420-8CBA-5765-B822-36006A36804F
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |53.220.87.28    |1372702404     |none                                    |DF7BDEE8-11A4-554C-9A46-BF678FFB92E8
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |146.189.104.94  |1372702489     |none                                    |E4C12778-5BB2-5169-A366-0AEAB7A196D0

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |nano32      |220.74.118.34   |simhostd1                  |LG                  |ai_3    |none       |A4BD6C79-A1B9-5D86-9E4F-42D5281BE5E7
|vm_34         |was                 |iron32      |158.222.100.139 |simhostb3                  |SUT                 |ai_3    |none       |2D84C44F-449D-54AA-8BD5-0DC5E1BCBF27
|vm_35         |db2                 |copper32    |164.226.250.199 |simhostc2                  |SUT                 |ai_3    |none       |6C7EA8C5-3809-5AF8-AAE1-EBAD900A6FC0
|vm_37         |client_daytrader    |nano32      |16.152.177.242  |simhostd3                  |LG                  |ai_4    |none       |972BC2A1-A750-507D-9BD3-5533076C80E2
|vm_36         |was                 |iron32      |24.146.188.108  |simhosta1                  |SUT                 |ai_4    |none       |FE0EE2A4-0845-55EC-A0D6-8A0065A3C07A
|vm_38         |db2                 |copper32    |89.82.252.146   |simhostc1                  |SUT                 |ai_4    |none       |6A3414F0-8D20-55DF-A9A4-36CED420DEC3
|vm_39         |netserver           |iron32      |133.61.53.184   |simhostc0                  |SUT                 |ai_5    |none       |A2C714DE-5001-5365-8F9E-6CFBD15410A3
|vm_40         |netclient           |platinum64  |53.220.87.28    |simhosta1                  |SUT                 |ai_5    |none       |4A65CF14-FEFD-539C-9AC1-6044CD63CC7C
|vm_47         |was                 |iron32      |49.4.184.107    |simhosta0                  |SUT                 |ai_8    |none       |C76ECBEC-7B88-586F-9915-B863622DF7ED
|vm_48         |lb                  |iron32      |20.123.111.87   |simhosta4                  |SUT                 |ai_8    |none       |E05D6232-BC52-54F2-8768-A5868E5FB4F1
|vm_50         |was                 |iron32      |236.129.55.114  |simhostb3                  |SUT                 |ai_8    |none       |4ABC2773-EDBE-5721-8598-27BB7AEB48CB
|vm_49         |client_daytrader    |nano32      |146.189.104.94  |simhostd0                  |LG                  |ai_8    |none       |AB302A64-2949-54F5-8837-C635E5F8909D
|vm_51         |db2                 |gold32      |243.57.239.247  |simhostb1                  |SUT                 |ai_8    |none       |99F2AE39-5F4D-5203-9CF7-BE8971E23335

The AI object ai_8, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |true                                 
|sut                                     |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |40.95M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11741267                      
|Total Connections Received                           |6492869                       
|Total Commands Processed                             |92968038                      
|Number of Keys                                       |545                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13967671296.0                 
|Data Size                                            |12135327276.0                 
|Index Size                                           |503085632                     
|Average Object Size                                  |6344.32041849                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |13                            
|SVMs                                                 |0                             
|AIs                                                  |4                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |13                            
|VMs ARRIVED                                          |48                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |4                             
|AIs ARRIVED                                          |8                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |186                           

 ---------------- VMC-WIDE COUNTERS ----------------
|35310FEB-1E59-53F1-A37C-0E4387DFF893 (simzone_a) VM RESERVATIONS|4                             
|DAEF899C-8C95-5B76-BE3F-58349C261E66 (simzone_d) VM RESERVATIONS|3                             
|8E18B916-CC95-566E-887E-3032BA7F835A (simzone_c) VM RESERVATIONS|3                             
|3076C295-3106-5ADC-8C71-D21083902D7A (simzone_b) VM RESERVATIONS|3                             

